code/control/solution/control.sol.cpp

```diff
@@ -0,0 +1,76 @@
+#include <cstdio>
+
+int isodd(int i) { return i % 2; }
```

Comment:
```suggestion
bool isodd(int i) { return i % 2; }
```

Reply:
I guess here we need to leave the int return type then.